presto-docs/src/main/sphinx/release/release-0.224.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Release 0.224
+=============
+
+General Changes
+---------------
+
+* Fix mis-swapped CPU time and scheduled time in query plan.
+* Fix correctness bug on queries containing nested coercible type cast since release 0.220.
+* Add support for recoverable grouped execution for writing unpartitioned table.
+* Add support for index-based access to row-type field with subscript.
+
+Security Changes
+----------------
+
+* Fix security issue with nextUri protocol
+
+Web UI Changes
+--------------
+
+* Add Planning Time in Query Details page
+
+Hive Connector Changes
+----------------------
+
+* Improve performance for bucketed table insertion when some buckets are empty.
+* Add config property ``hive.max-buckets-for-grouped-execution`` and session property ``max_buckets_for_grouped_execution`` to limit how many buckets a query can access and still take advantage of grouped execution.  If more than the configured number of buckets * partitions are queried, then the query will run without grouped execution.
```

Comment:
```
Add configuration property
```

Reply:
```
to limit the number of buckets a query can access while still taking the advantage
of grouped execution.
```

Reply:
Now sure about the last sentence, what is more than `buckets * partitions`?

Reply:
cc @rschlussel

Reply:
Maybe we don't need to mention buckets * partitions and we can just say buckets.  I was trying to indicate that it's about the total number of buckets accessed and not the number of buckets per partition, so  `the number of buckets per partition` times `the number of partitions`.  If a table is bucketed into 100 buckets and a user queries 10 partitions, then you're accessing 1000 buckets total.

Reply:
Agreed. Let's just say buckets here.